index.js

```diff
@@ -52,8 +53,9 @@ function adapter(uri, opts) {
 
   var prefix = opts.key || 'socket.io';
   var subEvent = opts.subEvent || 'messageBuffer';
-  var requestsTimeout = opts.requestsTimeout || 5000;
+  var requestsTimeout = opts.requestsTimeout || 1000;
   var withChannelMultiplexing = false !== opts.withChannelMultiplexing;
+  var consensusPercentage = isNaN(parseFloat(consensusPercentageFromEnv)) ? parseFloat(consensusPercentageFromEnv) : 1;
```

Comment:
```suggestion
  var consensusPercentage = !isNaN(parseFloat(consensusPercentageFromEnv)) ? parseFloat(consensusPercentageFromEnv) : 1;
```

Reply:
We also need to be careful with the terminology. It's written percentage, but actually, we are treating it as a number from 0 - 1. A Minor point that might create confusion while setting the value of the env variable

Reply:
done